src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -212,20 +245,36 @@ const ProjectExternalSubmissionsTable = ({
           id: projectId,
           formDefinitionIdentifier: formDefinitionIdentifier,
         }}
+        selectable='checkbox'
+        isRowSelectable={(s) =>
+          s.status === ExternalFormSubmissionStatus.New && !s.spam
+        }
         queryDocument={GetProjectExternalFormSubmissionsDocument}
         getColumnDefs={getColumnDefs}
         noData='No external form submissions'
         pagePath='project.externalFormSubmissions'
         recordType='ExternalFormSubmission'
         paginationItemName='submission'
         filters={filters}
-        handleRowClick={(row) => {
-          setSelectedId(row.id);
-          openFormDialog();
+        EnhancedTableToolbarProps={{
+          renderBulkAction: (selectedIds, selectedRows) => (
+            <LoadingButton
+              onClick={() => {
+                bulkUpdate({
+                  variables: {
+                    ids: selectedIds as string[],
+                  },
+                });
+              }}
+              loading={bulkLoading}
+            >
+              Convert {selectedRows.length} to Client
```

Comment:
this language isn't quite right, since some form types don't generate clients. Maybe `Bulk Review ({selectedRows.length}) Submissions`? (parens to match bulk services style)

Reply:
Yes much better, good point that it's not generic enough for all external forms. I also like this better because it's more consistent with the language we use inside the dialog when you are doing the review action for 1 individual submission.